packages/cli/src/commands/org/config/preview.ts

```diff
@@ -93,7 +75,49 @@ export default class Preview extends Command {
     return cfg.organization;
   }
 
+  private async handleInvalidSnapshot(snapshot: Snapshot) {
+    // TODO: CDX-362: handle invalid snashot cases
+    const {flags} = this.parse(Preview);
+    const pathToReport = snapshot.saveDetailedReport(flags.projectPath);
+    const report = snapshot.latestReport;
+
+    if (snapshot.requiresSynchronization()) {
+      cli.action.start('Synchronization');
+
+      const synchronizationPlanUrl = await this.getSynchronizationPage(
+        snapshot
+      );
+      this.warn(
+        dedent`Some conflicts were detected while comparing changes with the organization.
+        Click on the URL bellow to synchronize your organization before running the command again.
```

Comment:
```suggestion
        dedent`Some conflicts were detected while comparing changes between the snapshot and the target organization.
        Click on the URL below to synchronize your organization before running the command again.
```

Reply:
Tho I think @jpmarceau expert eye would be welcome here :D

Reply:
Building on Louis' suggestion: